ejb/ejb2-connector/ejb2-connector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/custom/connector/CallEJBStatefulBean.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * <p/>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.custom.connector;
+
+import org.apache.synapse.MessageContext;
+import org.wso2.carbon.connector.core.AbstractConnector;
+
+import java.lang.reflect.Method;
+
+public class CallEJBStatefulBean extends AbstractConnector {
+
+    @Override
+    public void connect(MessageContext messageContext) {
```

Comment:
It would be meaningful to inheritDoc from the parent and put a method description